src/lammpsio/data.py

```diff
@@ -394,6 +404,18 @@ def read(self):
                                 "Expected number of columns not read for mass"
                             )
                         masses[int(row[0])] = float(row[1])
+                elif "Bonds" in line:
+                    _readline(f, True)  # blank line
+                    bonds.typeid = numpy.zeros(bonds.N)
+                    bonds.members = numpy.zeros([bonds.N, 2])
+                    for i in range(int(bonds.N)):
+                        row = _readline(f, True).split()
+                        if len(row) < 4:
+                            raise IOError(
+                                "Expected number of columns not read for bonds"
+                            )
+                        bonds.typeid[i] = row[1]
+                        bonds.members[i, :] = row[2:]
```

Comment:
This should work fine:
```suggestion
                        bonds.members[i] = row[2:]
```